src/components/BaseButton.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+ <button type="button" :class="size" :style="{'background-color':backgroundColor}" >Trimite
+     <slot/>
+  </button>
+</template>
+
+<script>
+export default {
+  name: 'BaseButton',
+  props: {
+    size: {
+        type: String,
+        default: 'base-button'
+     },
+     backgroundColor: {
+       type: String,
+     }
+    }
+  }
+</script>
+
+<style >
+@baseButton component {
+  .base-button {
+    @apply inline-flex items-center px-4 py-2 border border-transparent text-base font-medium rounded-md shadow-sm text-white bg-indigo-600 active:bg-indigo-800 disabled:opacity-50;
+  }
+  .small-button {
+    @apply inline-flex items-center px-4 py-2 border border-transparent text-sm font-medium rounded-md shadow-sm text-white bg-indigo-600 active:bg-indigo-800 disabled:opacity-50;
```

Comment:
usually when there is a small button the paddings also change, hence the name small button 
![image](https://user-images.githubusercontent.com/54642532/129878884-3740d9f9-9d9d-451a-aae8-daf962f338c9.png)